packages/e2e/cypress/integration/ui/components/in-app-messaging/demo/demo.steps.ts

```diff
@@ -0,0 +1,31 @@
+import { Given, Then, When } from '@badeball/cypress-cucumber-preprocessor';
+
+Given('{string} checkbox is checked', (checkboxName: string) => {
+  cy.findByText(checkboxName)
+    .next()
```

Comment:
Why the `.next()`?

Reply:
`cy.findByText(checkboxName)` finds the span next to checkbox (label) and `.next()` finds the checkbox that we need to get the `data-checked` value.